poetry/core/masonry/builders/builder.py

```diff
@@ -273,6 +276,24 @@ def convert_entry_points(self):  # type: () -> dict
 
         return dict(result)
 
+    def convert_script_files(
+        self, base_path=None
```

Comment:
It looks like `base_path` as a parameter is not used anywhere else. For what use case would it be useful that this parameter exists?

It is probably fine to drop this for now so that the path resolution logic is more straightforward and only add it in once there is a clear use case.

Reply:
Unfortunately the bdist and the sdist builder has a difference as far as base path goes. That was the only way to solve it.

Reply:
My bad, this parameter was originally used by the bdist builder.